changelog/1.19.0.md

```diff
@@ -18,6 +18,13 @@ description: "Released on 05/19/2021"
   **Admin** > **Templates** no longer uses the following fields: **Project
   Repository URL** and **Project Git Service**
 
+### Deprecation notices 
+
+- helm: The `dashboard` deployment and service are being removed in a future
+  release. Any customers relying on routing rules outside the default ingress
+  included in the helm chart should remove any routing rules to this service
+  and direct traffic to the `cemanager` service instead.
```

Comment:
Did the embed stuff get in to 1.19? Otherwise IDK if we can deprecate this

Reply:
Lol I should be asking you that question

Reply:
There were a few embed things fixed post-this... so I don't think we should deprecate here.

Reply:
IMO we should close this PR (despite the beautiful CLOG additions) due to this:

https://www.notion.so/How-To-Ship-Customer-Visible-Changes-477b63d362aa400ba9075a01b09084bd

Reply:
LMK if I'm off-base with that thought - but basically this isn't something we document until beta stage